release.md

```diff
@@ -121,3 +126,16 @@ Per the instructions [here](https://conda-forge.org/docs/maintainer/updating_pkg
     - Select *Bot commands* and put the following title (change `username`): `@conda-forge-admin, please add user @username`
 2. A PR will be auto-created on the repo, and will need to be merged by an existing maintainer.
 3. The new user will need to **check their email for an invite link to click**, which should be https://github.com/conda-forge
+
+## Troubleshooting failed Github actions
+
+If some Github actions are failing and they aren't automatically ran in the PR request, to easily test them without having to push to main:
```

Comment:
Mega grammar nit: `and they aren't automatically run in the PR request` -> `and they aren't automatically run in the PR's automated tests`